packages/web/modals/increase-concentrated-liquidity.tsx

```diff
@@ -120,12 +122,35 @@ export const IncreaseConcentratedLiquidityModal: FunctionComponent<
     <ModalBase
       {...props}
       isOpen={props.isOpen && showModalBase}
-      title={t("clPositions.increaseLiquidity")}
-      className="!max-w-[500px]"
+      hideCloseButton
+      className="max-h-[95vh] !max-w-[500px] overflow-auto"
     >
+      <div className="align-center relative mb-8 flex flex-row">
+        <div className="absolute left-0 flex h-full items-center text-sm" />
+        <h6 className="flex-1 text-center">
+          {t("clPositions.increaseLiquidity")}
+        </h6>
+        <div className="absolute right-0">
+          <IconButton
+            aria-label="Close"
+            mode="unstyled"
+            size="unstyled"
+            className="!p-0"
+            icon={
+              <Icon
+                id="close-thin"
+                className="text-wosmongton-400 hover:text-wosmongton-100"
+                height={24}
+                width={24}
+              />
+            }
+            onClick={props.onRequestClose}
+          />
+        </div>
+      </div>
```

Comment:
i think the basemodal provides a close button for use if we simply set hideclosebutton=false